app/views/notes/index.html.erb

```diff
@@ -1,13 +1,27 @@
 <% notes.each_with_index do |note, i| %>
+
+
+
   <% if authorized?(note) %>
     <h3>
       Note <%= i + 1 %>
     </h3>
-    <div>Name: <%= note.note_with_type %></div>
-    <div>Author: <%= note.user_id %></div>
-    <div>Category: <%= note.category.name %></div>
-    <div>Body: <%= note.body %></div>
+    <div class="row">
+      <div class="card blue-grey file-card col m8 offset-m2">
```

Comment:
Same as the other note, we need to establish the look on smaller screens too please